README.md

```diff
@@ -56,6 +56,7 @@ import {
     SolletExtensionWalletAdapter,
     SolletWalletAdapter,
     TorusWalletAdapter,
+    KeystoneWalletAdapter,
```

Comment:
Please limit changes to adding the package only. We don't add all new wallets to the example or starter projects.